extensions/spring-data-jpa/deployment/src/main/java/io/quarkus/spring/data/deployment/SpringDataJPAProcessor.java

```diff
@@ -53,6 +55,27 @@ void build(CombinedIndexBuildItem index,
             BuildProducer<GeneratedBeanBuildItem> generatedBeans,
             BuildProducer<AdditionalBeanBuildItem> additionalBeans) {
 
+        Config config = ConfigProvider.getConfig();
+        Optional<String> springJpaConfig = config.getOptionalValue("spring.jpa.show-sql", String.class);
+        Map<String, String> springJpaPropertiesMap = new HashMap<>();
+        springJpaPropertiesMap.put("spring.jpa.show-sql", "quarkus.hibernate-orm.log.sql");
+        springJpaPropertiesMap.put("spring.jpa.properties.hibernate.dialect ", "quarkus.hibernate-orm.dialect");
+        springJpaPropertiesMap.put("spring.jpa.properties.hibernate.dialect.storage_engine",
+                "quarkus.hibernate-orm.dialect.storage-engine");
+        springJpaPropertiesMap.put("spring.jpa.generate-ddl", "quarkus.hibernate-orm.database.generation");
+
+        Iterable<String> iterablePropertyNames = config.getPropertyNames();
+        List<String> propertyNames = new ArrayList<String>();
+        iterablePropertyNames.forEach(propertyNames::add);
+        List<String> springProperties = propertyNames.stream().filter(s -> s.matches("spring.jpa.*")).collect(toList());
+        if (!springProperties.isEmpty()) {
+            for (String springProperty : springProperties) {
+                LOGGER.warn("Quarkus does not support the " + springProperty
+                        + " you may try to use the quarkus equivalent : " + springJpaPropertiesMap.get(springProperty));
```

Comment:
This will be null if you have a `spring.jpa.` property that is not defined in your map.